tools/scripts/aducm.mk

```diff
@@ -162,6 +162,24 @@ aducm3029_run: all
 		-s "$(ADUCM_DFP)/openocd/scripts" -f target/aducm3029.cfg \
 		-c "program  $(BINARY) verify reset exit" $(HIDE)
 
+.PHONY: $(BINARY).gdb
+$(BINARY).gdb:
+	@echo target remote localhost:3333 > $(BINARY).gdb
+	@echo load $(BINARY) >> $(BINARY).gdb
+	@echo file $(BINARY) >> $(BINARY).gdb
+	@echo monitor reset halt >> $(BINARY).gdb
+	@echo tb main >> $(BINARY).gdb
+	@echo c >> $(BINARY).gdb
+	@echo refresh >> $(BINARY).gdb
+
+.PHONY: debug
+debug: all $(BINARY).gdb
+	(openocd -s "$(OPENOCD_SCRIPTS)" -f interface/cmsis-dap.cfg \
+		-s "$(ADUCM_DFP)/openocd/scripts" -f target/aducm3029.cfg \
+		-c "init" &);
+	/usr/bin/arm-none-eabi-gdb -tui --command=$(BINARY).gdb || \
```

Comment:
this doesn't seems to be a path that work on windows :)
arm-none-eabi-gdb should be enough

Reply:
look at the PR  comment, it's explained there

Reply:
the problem is that the one that comes with the toolchain doesn't provide -tui, so I put this in-place until they wil ship one with the -tui option
it won't fail on windows

Reply:
So actually i realised it will fail on windows but not for that reason.
& (launch as separate process) and || (or operator) are actually bash operators, this is what will not work on windows after all :) I am changing the commit message accordingly.

I think we should add this stuff gradually - first have make debug on linux, then extend to windows.
Is that ok with you ?